rfcs/text/0000-alternative-planning.md

```diff
@@ -0,0 +1,315 @@
+detailed quarter by quarter objectives as detailed in [Human Cell Atlas DCP: Strategy](https://docs.google.com/document/d/1hBsj2BB9SjuJHodS3hM7cRJzXc5x-uYdOQL3LtGjYWM). 
+Project leads set and review the aims and indicators in their role as leaders of the project. Detailed planning is still very much
+necessary and desirable, but this responsibility is fulfilled by product owners working in conjunction with tech leads
+and developers. The quarter by quarter roadmap still exists but is more theme than detail based, and is owned solely
+by product owners aided by project managers. Success is judged on how well we meet the indicators over time, *not* by whether the DCP fulfilled a specific
+objective.
+
+This is to preserve agility - the ability to adapt objectives over short timeframes, as short as days or even less. This is for 
+[core agile reasons](https://agilemanifesto.org/); individuals and interactions, working software over documentation, 
+customer collaboration over contract negotiation, responding to change over following a plan. Especially in this 
+project, we uncover new information all the time, we realize on contact with constantly evolving scientific reality that 
+our plans won't work, we realize that actually this thing has high and immediate importance compared to this other thing.
+
+I think that the important thing here is communicating these changes and shifting priorities to everyone else.
+Following a plan that dissolves on contact with reality should be secondary. Managing by indicators and broad aims
+rather than detailed tasks allows us to adapt *and* gives us yardsticks to measure whether the DCP is really delivering
+user value or not. It removes the perceived need (whether true or not, perception is reality) for any change to require
+consultation and associated delay up to project leads and back, which acts as a strong drag on decision making.
+
+Project leads can still act as product owners but need to be very clear, both to themselves and others, when they are
+wearing this hat and when they are wearing the product owner hat. When they're acting in their product owner role
+they should be careful not to overrule other product owners whether consciously or not. 
+
+Naturally, aims and indicators are likely not a perfect solution. Whilst they have pros:
+
+1) Allows teams to adapt quickly to changing circumstances without a formal planning cycle and communication up and down
+the planning stack, as long as the metric is still valid.
+
+They also have cons:
+
+1) Metrics themselves can become a point of inflexibility. There needs to be a mechanism by which bad metrics can be
+recognized and changed.
+2) It may not be possible to create metrics for some objectives. This could mean the objective is not clear enough.
+
+Nonetheless, I believe that management by aims and indicators is a management framework that it would be very valuable
+to try.
+
+#### Theme teams
```

Comment:
Side note: This is not incompatible with @brianraymor's planning RFC. We could do both!

Reply:
Agreed.